nexus-sdk-js/src/Resource/index.ts

```diff
@@ -0,0 +1,73 @@
+import { Context, Distribution } from '../utils/types';
+import {
+  ResourceMetadataKeys,
+  ResourceResponseCommon,
+  ResourceResponse,
+  ListResourceResponse,
+} from './types';
+
+export {
```

Comment:
we should only export stuff in `Nexus.ts`.

I'm planning on adding a index.ts that will export everything that needs exported, kind of like these guys https://github.com/ant-design/ant-design/blob/master/components/index.tsx